### PR TITLE
[RHDEVDOCS-4231] add tkn-pac info

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): enterprise-4.11 and later

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4231

Link to docs preview:
https://49582--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html

Additional information: Follow-up from this PR: https://github.com/openshift/openshift-docs/pull/48302

### CODE REVIEW

**Review by concaf — APPROVED** [review]

File: modules/op-installing-tkn-on-macos.adoc [hunk 1]
```diff
@@ -12,11 +12,15 @@ For macOS, the `tkn` CLI is provided as a `tar.gz` archive.
...
+. Download the relevant CLI.
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-macos-amd64.tar.gz[macOS]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-macos-arm64.tar.gz[macOS on ARM]
 
 . Unpack and unzip the archive.
 
-. Move the `tkn` binary to a directory on your PATH.
+. Move the `tkn` and `tkn-pac` binaries to a directory on your PATH.
```
Comment (rolfedh) [inline modules/op-installing-tkn-on-macos.adoc:23]:
```suggestion
. Move the `tkn` and `tkn-pac` binaries to a directory on your `PATH`.
```
[Contributor's guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference): 
Use backticks for all non-GUI "system items", including:
Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input

File: modules/op-installing-tkn-on-macos.adoc [hunk 2]
```diff
@@ -12,11 +12,15 @@ For macOS, the `tkn` CLI is provided as a `tar.gz` archive.
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-macos-amd64.tar.gz[CLI].
+. Download the relevant CLI.
```
Comment (rolfedh) [inline modules/op-installing-tkn-on-macos.adoc:15]:
```suggestion
. Download the relevant CLI tool.
```
Maybe call this a "CLI tool"?

**Review by rolfedh — APPROVED** [review]
Nice work, Silke!

**Review by ppitonak — APPROVED** [review]

**Review by rolfedh — COMMENTED** [review]
Where needed, you can insert the following note that the feature is in TP.
```
:FeatureName: Running {pipelines-title} on ARM hardware
include::snippets/technology-preview.adoc[]
```

**Review by rolfedh — APPROVED** [review]

**Pull request merged**